### PR TITLE
Add docker example

### PR DESCRIPTION
**Pull request by weissreto**

### CODE REVIEW

**Comment by ivy-rew** [comment]
add to 'pom.xml' in the root of this project ... so that it is being built + raise-version.sh can work on it.

File: docker/pom.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	<groupId>ch.ivyteam.ivy.project.demo.ci.docker</groupId>
+	<artifactId>ivy-application</artifactId>
+	<version>9.3.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<scm>
+		<connection>scm:git:git@github.com:axonivy/project-build-examples.git</connection>
+	</scm>
+
+	<modules>
```
Comment (ivy-rew) [inline docker/pom.xml:12]: this build doesn't run on my machine, due to version clashes it fails when building the app
```
[INFO] --- maven-assembly-plugin:3.1.0:single (create.app) @ ivy-application ---
[INFO] Reading assembly descriptor: assembly.xml
[INFO] artifact ch.ivyteam.ivy.project.demo.ci.deploy.application:base: checking for updates from central
[INFO] Failure detected.
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for ivy-application 9.3.0-SNAPSHOT:
[INFO] 
[INFO] base ............................................... SUCCESS [ 46.808 s]
[INFO] solution ........................................... SUCCESS [  0.890 s]
[INFO] customer ........................................... SUCCESS [  0.674 s]
[INFO] ivy-application .................................... FAILURE [  1.045 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  51.740 s
[INFO] Finished at: 2021-12-21T16:19:44+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.1.0:single (create.app) on project ivy-application: Failed to create assembly: Unable to resolve dependencies for assembly 'app': Failed to resolve dependencies for assembly: No versions are present in the repository for the artifact with a range [9.3.0-SNAPSHOT,)
[ERROR]   ch.ivyteam.ivy.project.demo.ci.deploy.application:base:iar:null
[ERROR] 
[ERROR] from the specified remote repositories:
[ERROR]   central (https://repo.maven.apache.org/maven2, releases=true, snapshots=false)
[ERROR] Path to dependency: 
[ERROR] 	1) ch.ivyteam.ivy.project.demo.ci.docker:ivy-application:pom:9.3.0-SNAPSHOT
[ERROR] 	2) ch.ivyteam.ivy.project.demo.ci.deploy.application:solution:iar:9.3.0-SNAPSHOT
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :ivy-application

```

**Comment by ivy-rew** [comment]
nice instructions, nice and mature example ... neatly documented :+1: :1st_place_medal: 
just include it in a build, fix the pom and the PR should be fine

**Review by ivy-rew — APPROVED** [review]

File: docker/pom.xml (hunk 1 above)
Comment (ivy-rew) [reply docker/pom.xml:12]: my bad, works with 'mvn clean install'

Comment (weissreto) [reply docker/pom.xml:12]: Projects have same maven coordinates as the ones in deploy. Have changed them all

**Pull request merged**